src/net_processing.cpp

```diff
@@ -2804,6 +2849,15 @@ void PeerManagerImpl::ProcessMessage(CNode& pfrom, const std::string& msg_type,
             if (fReachable)
                 vAddrOk.push_back(addr);
         }
+        peer->m_addr_processed += num_proc;
+        peer->m_addr_rate_limited += num_rate_limit;
+        LogPrint(BCLog::NET, "Received addr: %u addresses (%u processed, %u rate-limited) from peer=%d%s\n",
+                 vAddr.size(),
+                 num_proc,
+                 num_rate_limit,
+                 pfrom.GetId(),
+                 fLogIPs ? ", peeraddr=" + pfrom.addr.ToString() : "");
```

Comment:
Is there a reason to log the ip address outside of the version handshake? This seems redundant, as the ip shouldn't change afterwards and should be easy to lookup by grepping for the handshake of the peer id.

Otherwise this encourages to put other redundant stuff (ua-string, ...) into the log messages as well.

Reply:
Proposed in #22604.